### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

**Comment by mcovarr** [comment]
_adding_ tests?  has @geoffjentry's GitHub been hacked? 😛

**Comment by mcovarr** [comment]
I assume mostly what's happening here is a CromwellTestkitSpec -> Centaur conversion of existing tests.  Perhaps this is an issue with the original tests, but it would be nice if there were some comments as to what exactly is being tested.  Sometimes it's clear from the names of the tests and/or WDL, but I'm not sure I fully appreciate all the subtleties in all of them.

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/104/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by geoffjentry** [comment]
Your assumption is correct. Truth be told I often have to spend a while figuring them out myself (as I often need to slightly change them, and I'm also looking for dupes), so your point is well taken

File: src/main/resources/standardTestCases/input_localization.test [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: input_localization
+testFormat: workflowsuccess
+backend: Local
```
Comment (geoffjentry) [inline src/main/resources/standardTestCases/input_localization.test:6]: Need to add this to workflow options

**Comment by kshakir** [comment]
I like @mcovarr's suggestion of a sprinkling of comments, then 👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/104/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**